### DISCUSSION TITLE
Order edit - adding/edit adjustments on carts

### DISCUSSION THREAD

**Discussion by samuelbirch** [craftcms/commerce#2440] [Ideas]
Can the order edit adjustments be available when a new order is created in the CP?

Currently they are only available when an order is complete.

Latest version on commerce.

**Comment by samuelbirch** [comment]
@nfourtythree are there any updates on this issue? - cheers

**Comment by lukeholder** [comment]
@samuelbirch 

This is not a bug but a feature at the moment :)

Order adjustments editing is not available while the order is a cart, as the customer is still able to edit the cart, and we don't have the concept of a cart being locked for dynamic recalculation, so any manual edits to adjustments would be lost when the cart is recalculated. You also can't edit the price of the line item as well for the same reason.

We would need to introduce some type of 'Draft' order mode, that does not complete the order, but is not a cart that will recalculate. This would be a large change to the system at the moment which we can't add in the short term.

A work around could be that you make an order status called Draft that is the new default status for a completed order that was made from the `origin` of the CP, and only used by store admins when completing creating new orders so that adjustments can be edited.

 This Draft status wouldn't trigger any emails to the customers. Then when you have finished making the order, you could move it your previous default status that would trigger emails to the customer.

Then on the front-end when a custom completes an cart, they get the default status, but the store admins get another.

```
use craft\commerce\Plugin;
use craft\commerce\events\DefaultOrderStatusEvent;
use craft\commerce\services\OrderStatuses;
use craft\commerce\models\OrderStatus;
use craft\commerce\elements\Order;
use yii\base\Event;

Event::on(
    OrderStatuses::class,
    OrderStatuses::EVENT_DEFAULT_ORDER_STATUS,
    function(DefaultOrderStatusEvent $event) {
        // @var Order $order
        $order = $event->order;

       if($order->origin == 'cp'){ // if the order was created in the control panel (could also check an order custom field if you want ot manually control this in the CP)
        $event->orderStatus = Plugin::getInstance()->getOrderStatuses()->getOrderStatusByHandle('draft');
       }
    }
);
```

Will keep this ticket open as a feature request.

**Comment by samuelbirch** [comment]
@lukeholder Thanks for the update.
